main.go

```diff
@@ -46,4 +70,12 @@ func main() {
 			fmt.Printf("Current: %s; Latest: %s", version.Current(), newVersion)
 		}
 	}
+
+	return 0
+}
+
+func flushSentry() {
+	fmt.Print("Sending error report ...")
+	sentry.Flush(60 * time.Second)
+	fmt.Printf(" done, thanks you %s\n", color.RedString("❤️"))
```

Comment:
Should be ` done, thank you %s\n` or ` done, thanks %s\n`

Reply:
Fixed